src/practitest_firecracker/core.clj

```diff
@@ -50,13 +50,13 @@
 
             "create-and-populate-testset"
             (do
+              (println "Start Running Firecracker, Version: " fc-version)
```

Comment:
Is there a reason to use `println` other than `log/info` or something like this ?

Reply:
no reason, will change to log/info